examples/transaction/transfer_transaction_gigabar.py

```
@@ -0,0 +1,127 @@
+    try:
+        tx = (
+            AccountCreateTransaction()
+            .set_key(recipient_key.public_key())
+            .set_initial_balance(Hbar.from_tinybars(0))
+        )
+        response = tx.freeze_with(client).sign(operator_key).execute(client)
+        receipt = response.get_receipt(client)
+
+        recipient_id = receipt.account_id
+        print(f"✅ Success! Created a new recipient account with ID: {recipient_id}")
+        return recipient_id, recipient_key
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)